wp-cache.php

```diff
@@ -2085,8 +2085,16 @@ function wp_cache_debug_settings() {
 	if ( ! isset( $wp_cache_debug_log ) || $wp_cache_debug_log == '' ) {
 		extract( wpsc_create_debug_log() ); // $wp_cache_debug_log, $wp_cache_debug_username
 	}
-	$log_file_link = "<a href='" . site_url( str_replace( ABSPATH, '', "{$cache_path}view_{$wp_cache_debug_log}" ) ) . "'>$wp_cache_debug_log</a>";
-	$raw_log_file_link = "<a href='" . site_url( str_replace( ABSPATH, '', "{$cache_path}{$wp_cache_debug_log}" ) ) . "'>" . __( 'RAW', 'wp-super-cache' ) . "</a>";
+        if ( function_exists( 'wp_normalize_path' ) ) {
+		$cache_url = wp_normalize_path( $cache_path );
+		$cache_url = str_replace( wp_normalize_path( ABSPATH ), '', $cache_url );
+	} else {
+		$cache_url = str_replace( '\\', '/', $cache_path );
+		$cache_url = str_replace( str_replace( '\\', '/', ABSPATH ), '', $cache_url );        
+	}
+	$cache_url = site_url( $cache_url );
```

Comment:
```suggestion
	$cache_url = trailingslashit( site_url( $cache_rel_path ) );
```
It's possible to realpath strips last slash.